src/main/scala/org/ergoplatform/modifiers/history/BlockTransactions.scala

```diff
@@ -46,7 +46,7 @@ case class BlockTransactions(headerId: ModifierId, txs: Seq[ErgoTransaction])
 object BlockTransactions {
   val modifierTypeId: ModifierTypeId = ModifierTypeId @@ (102: Byte)
 
-  def rootHash(ids: Seq[ModifierId]): Digest32 = Algos.merkleTreeRoot(LeafData @@ ids)
+  def rootHash(ids: Seq[ModifierId]): Digest32 = Algos.merkleTreeRoot(LeafData @@ ids.map(idToBytes))
```

Comment:
Potentially slow operation: here we will do hundreds or thousands conversions per block.

Reply:
Use serializedId?